### PR TITLE
feat(errors): Add ResourceExpiredError

### PR DESCRIPTION
**Pull request by remyleone**

### CODE REVIEW

File: scw/errors.go [hunk 1]
```diff
@@ -339,3 +340,18 @@ func (e ConfigFileNotFoundError) IsScwSdkError() {}
...
+type ResourceExpiredError struct {
+	Resource     string    `json:"resource"`
+	ResourceID   string    `json:"resource_id"`
+	ExpiredSince time.Time `json:"expired_since"`
+
+	RawBody json.RawMessage `json:"-"`
+}
+
+func (r ResourceExpiredError) Error() string {
+	return fmt.Sprintf("scaleway-sdk-go: resource #{r.Resource} with ID #{r.ResourceID} expired since %s.", r.ExpiredSince.String())
```
Comment (kindermoumoute) [inline scw/errors.go]: > resource #{r.Resource} 
Is this a Python notation? 🤔

File: scw/errors.go [hunk 2]
```diff
@@ -339,3 +340,18 @@ func (e ConfigFileNotFoundError) IsScwSdkError() {}
...
+type ResourceExpiredError struct {
+	Resource     string    `json:"resource"`
+	ResourceID   string    `json:"resource_id"`
+	ExpiredSince time.Time `json:"expired_since"`
+
+	RawBody json.RawMessage `json:"-"`
+}
+
+func (r ResourceExpiredError) Error() string {
+	return fmt.Sprintf("scaleway-sdk-go: resource %s with ID %s expired since %s.", r.Resource, r.ResourceID, r.ExpiredSince.String())
```
Comment (kindermoumoute) [inline scw/errors.go]: nit: no dot at the end

File: scw/errors.go [hunk 3]
```diff
@@ -339,3 +340,18 @@ func (e ConfigFileNotFoundError) IsScwSdkError() {}
 func (e ConfigFileNotFoundError) Error() string {
 	return fmt.Sprintf("scaleway-sdk-go: cannot read config file %s: no such file or directory", e.path)
 }
+
+// ResourceExpiredError implements the SdkError interface
+type ResourceExpiredError struct {
```
Comment (kindermoumoute) [inline scw/errors.go]: the error must be added in `unmarshalStandardError`

Comment (remyleone) [reply scw/errors.go]: done

**Pull request merged**